hash_practice/exercises.py

```diff
@@ -1,19 +1,43 @@
 
+from ntpath import join
+from sqlalchemy import true
+
+
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n * m log m)
+        ^ Not sure on the time complexity. Here, n represents the number of words in the 
+        input string, while m represents the length of an individual word. 

```

Comment:
This is correct!

However, we can make a simplifying assumption. Since we know the words are English words and English words do not get too long (about 5 letters per word on average), the effect they have on this algorithm is dwarfed by the number of words in the list (which could easily get in the hundreds and thousands for a big list).

Thus, we can just say this is O(n).